### DISCUSSION TITLE
Get all authenticated users at any given time

### DISCUSSION THREAD

**Discussion by 0324bence** [pocketbase/pocketbase#2521] [Q&A]
I would like to periodically check properties of all users who are currently using the site and send events based on the results. However I could not find a way for me to determine if a user is authenticated or to get a list of them.

Is there any way for me, to get a list of currently authenticated users at a given time?

**Comment by ganigeorgiev** [comment]
> Is there any way for me, to get a list of currently authenticated users at a given time?

No (at least not out of the box). PocketBase web APIs are stateless, meaning that there are no auth sessions in the traditional sense (even the generated auth tokens are not stored in the database).

A single user is considered authenticated as long as they send a request with valid `Authorization: TOKEN` header.
